### PR TITLE
Fixed loader/doc issues w.r.t. loadId

### PR DESCRIPTION
**Pull request by shreyas-damle**
- Fixed loader/doc issues w.r.t. loadId
- Moved commit before pdf generation
- Updated query filter, to get latest id

### CODE REVIEW

File: pebblo/app/service/loader/loader_doc_service.py [hunk 1]
```diff
@@ -310,6 +310,9 @@ def process_request(self, data):
                 )
                 loader_response_output.append(doc_obj)
 
+            # Commit before generating report, so that even if report generation fails then values would be in db.
+            self.db.session.commit()
+
             if self.data["loading_end"]:
```
Comment (gr8nishan) [inline pebblo/app/service/loader/loader_doc_service.py:316]: can we do it in get

**Review by gr8nishan — APPROVED** [review]

File: pebblo/app/service/loader/loader_doc_service.py (hunk 1 above)
Comment (shreyas-damle) [reply pebblo/app/service/loader/loader_doc_service.py]: Fixed.

File: pebblo/app/service/loader/loader_doc_service.py [hunk 2]
```diff
@@ -310,7 +310,10 @@ def process_request(self, data):
                 )
                 loader_response_output.append(doc_obj)
 
-            if self.data["loading_end"]:
+            # Commit before generating report, so that even if report generation fails then values would be in db.
+            self.db.session.commit()
+
+            if self.data.get("loading_end", False):
```
Comment (gr8nishan) [inline pebblo/app/service/loader/loader_doc_service.py:316]: no need to do False even if None return False

File: pebblo/app/service/local_ui/loader_apps.py [hunk 3]
```diff
@@ -206,15 +206,21 @@ def get_all_loader_apps(self):
             _, ai_loader_apps = self.db.query(table_obj=AiDataLoaderTable)
 
             # Preparing all loader apps
+            app_processed = list()
             all_loader_apps: list = []
             for loader_app in ai_loader_apps:
                 app_data = loader_app.data
                 if app_data.get("docEntities") not in [None, {}] or app_data.get(
```
Comment (gr8nishan) [inline pebblo/app/service/local_ui/loader_apps.py:213]: we can just do  if app_data.get("docEntities")  or app_data.get("docTopics")

Comment (shreyas-damle) [reply pebblo/app/service/local_ui/loader_apps.py:213]: This is to skip processing even if add_data = {"docEntities": None} or app_data = {"docEntities": {}} or app_data = {}

File: pebblo/app/service/loader/loader_doc_service.py (hunk 2 above)
Comment (shreyas-damle) [reply pebblo/app/service/loader/loader_doc_service.py:316]: We always get `loading_end` either `True` or `False`

File: pebblo/app/service/local_ui/loader_apps.py (hunk 3 above)
Comment (shreyas-damle) [reply pebblo/app/service/local_ui/loader_apps.py]: Fixed.

File: pebblo/app/service/loader/loader_doc_service.py (hunk 2 above)
Comment (shreyas-damle) [reply pebblo/app/service/loader/loader_doc_service.py]: Fixed.

**Review by gr8nishan — APPROVED** [review]

**Pull request merged**